expansion/src/main/java/no/unit/nva/expansion/ResourceExpansionServiceImpl.java

```diff
@@ -100,11 +100,9 @@ public ExpandedDataEntry expandEntry(Entity dataEntry, boolean useUriContext) th
     @Override
     public ExpandedOrganization getOrganization(Entity dataEntry) throws NotFoundException {
         if (dataEntry instanceof TicketEntry ticketEntry) {
-
-            var organizationId = nonNull(ticketEntry.getResponsibilityArea())
-                                      ? ticketEntry.getResponsibilityArea()
-                                      : resourceService.getResourceByIdentifier(ticketEntry.getResourceIdentifier())
-                                          .getResourceOwner().getOwnerAffiliation();
+            var organizationId = Optional.ofNullable(ticketEntry.getReceivingOrganizationDetails())
+                                     .map(ReceivingOrganizationDetails::subOrganizationId)
+                                     .orElse(getAffiliationFromResourceOwner(ticketEntry));
```

Comment:
Optional.ofNullable(ticketEntry.getReceivingOrganizationDetails())
                                     .map(ReceivingOrganizationDetails::subOrganizationId)
                                     .or(() -> ticketEntry.getReceivingOrganizationDetails().topLevelOrganizationId())
                                     .orElse(getAffiliationFromResourceOwner(ticketEntry));
                                     
                                     Maybe try to send first to receiving top level org